matchzoo/losses/zrank_losses.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import zoo.pipeline.api.autograd as A
+from zoo.pipeline.api.keras.layers import *
+from keras.utils.generic_utils import deserialize_keras_object
+
+mz_specialized_losses = {'rank_hinge_loss', 'rank_crossentropy_loss'}
+
+def serialize(rank_loss):
+    return rank_loss.__name__
+
+
+def deserialize(name, custom_objects=None):
+    return deserialize_keras_object(name,
+                                    module_objects=globals(),
+                                    custom_objects=custom_objects,
+                                    printable_module_name='loss function')
+
+def rank_hinge_loss(kwargs=None):
```

Comment:
Wondering would it be possible if we only simply add a new method `z_rank_hinge_loss`  in `rank_losses`

Reply:
is this useless as you've defined it in `zmain.py` ?